server/utils/url-utils.js

```diff
@@ -0,0 +1,29 @@
+const isDev = process.env.NODE_ENV !== 'production';
+const isBeta = !!process.env.BETA;
+
+export function getEmailSender() {
+  return process.env.EMAIL_SENDER || 'team@freecodecamp.com';
+}
+
+export function getPort() {
+  return isDev ? (process.env.SYNC_PORT || 3000) : 443;
```

Comment:
nit: use a if/esle condition for readability here. 
fix: if not dev port should be omitted.